v2/yarpcthrift/thriftrw-plugin-yarpcv2/roundtrip_test.go

```diff
@@ -22,43 +22,39 @@ package main_test
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/atomic"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/atomic/readonlystoreclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/atomic/readonlystoreserver"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/atomic/storeclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/atomic/storeserver"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/baseserviceclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/baseserviceserver"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/emptyserviceclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/emptyserviceserver"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/extendemptyclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/extendemptyserver"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/extendonlyclient"
-	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpc/internal/tests/common/extendonlyserver"
+	"go.uber.org/yarpc/v2/internal/internaltesttime"
+	"go.uber.org/yarpc/v2/yarpchttp"
+	"go.uber.org/yarpc/v2/yarpcrouter"
+	"go.uber.org/yarpc/v2/yarpcthrift"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic/readonlystoreclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic/readonlystoreserver"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic/storeclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/atomic/storeserver"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/baseserviceclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/baseserviceserver"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/emptyserviceclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/emptyserviceserver"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/extendemptyclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/extendemptyserver"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/extendonlyclient"
+	"go.uber.org/yarpc/v2/yarpcthrift/thriftrw-plugin-yarpcv2/internal/tests/common/extendonlyserver"
 )
 
 func TestRoundTrip(t *testing.T) {
 	tests := []struct{ enveloped, multiplexed bool }{
 		{true, true},
-		// Skipping for now until flaky test fixed.
-		// Uncomment this when fixed.
-		// https://github.com/yarpc/yarpc-go/issues/1171
-		//{true, false},
+		{true, false},
```

Comment:
<3

Reply:
Make sure to run this test a few times with `go test count=1` so test results aren't cached.